hack/Dockerfile

```diff
@@ -51,25 +51,18 @@ RUN set -ex \
 	&& add-apt-repository --no-update --yes "deb http://archive.ubuntu.com/ubuntu/ jammy multiverse" \
 	&& add-apt-repository --no-update --yes "deb http://archive.ubuntu.com/ubuntu/ jammy-security universe" \
 	&& add-apt-repository --no-update --yes "deb http://archive.ubuntu.com/ubuntu/ jammy-updates multiverse" \
+	&& curl -sS https://deb.nodesource.com/setup_20.x | bash - \
```

Comment:
You also need to upgrade [the CI job ](https://github.com/artefactual/archivematica/blob/155f229fa7a15cc7f01a57eb5a9fc0335a20f5c8/.github/workflows/test.yml#L84).

Reply:
sure. Thanks!